gui/build.rs

```diff
@@ -8,6 +9,30 @@ const LINUX_INCLUDE: &str = r#"
 "#;
 
 fn main() {
+    if std::env::var("CARGO_FEATURE_GUI_SLINT").is_ok() {
```

Comment:
There should be a better way than this. Feature flag should be used to enable additional code instead of using like this.

Reply:
I tried, but AFAIK:
1. there can be only one build script per crate.
2. There is no way to tell if we are compiling the library or the binary.

Reply:
Have you try `CARGO_BIN_NAME`?

Reply:
oh :no_mouth:

Reply:
I tried CARGO_BIN_PATH, but it's not set :(.

Reply:
Does Slint support including those files in Rust instead of building it here? Something like `include_slint!("file.slint").

Reply:
I don't think so. The only other option is including slint source directly in Rust, there's a macro for that

Reply:
Another idea is putting the FFI for Qt behind a feature flag and enabling it from CMake.

Reply:
I did that, take a look